internal/testutil/env.go

```diff
@@ -61,3 +61,25 @@ func Chdir(t TestingT, dir string) string {
 
 	return wd
 }
+
+func InsertPathEntry(t TestingT, path string) {
```

Comment:
"Prepend" better captures the intended effect over "Insert" (i.e. insert where?)

Reply:
Agree, it does. I removed this function completely though :)